DocsCode/DDSCircleLayerTutorial/DDSCircleLayerTutorialViewController.swift

```diff
@@ -3,60 +3,44 @@ import Mapbox
 
 class DDSCircleLayerTutorialViewController: UIViewController, MGLMapViewDelegate {
     
-    // #-code-snippet: dds-circle initialize-map-swift
```

Comment:
@captainbarbosa Are these `#-code-snippet:` bits no longer necessary?

Reply:
They're coming back with new names in 254c869 😄